access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable an organization administrator to manage how user accounts and service accounts are permitted to interact with Kafka resources, such as instances and topics. A user account enables users in your organization to access resources. A service account enables your application or tool to connect securely to your resources.
+
+Administrators can allow or deny operations that specific accounts or that all accounts in the organization can perform for Kafka resources. A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. Administrators can also allow or deny access for other users to define account permissions for an instance.
+
+Users who aren't administrators can fully access a Kafka instance that they create and can manage the level of access for service accounts that they create for their own instances. However, only administrators can manage the level of access of other users or of service accounts created by other users for a Kafka instance.
+
+Administrators can define ACLs for the following components:
+
+* A single account or all accounts within the organization
+* A single named resource, all resources of a particular type (such as a topic, consumer group, or transactional ID), or all resources of a particular type with a specified prefix
+* A single operation (such as `Write`) for a resource or all operations for a resource
+
+If two or more permission settings in an ACL match a request being made to the Kafka broker and at least one of the matching permissions specifies that the action is denied, then the request is denied.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+=== Supported ACL permissions in {product}
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to one or more Kafka APIs or request types that are applicable for the specified resource.
+
+For example, a `Read` operation for a `Topic` resource corresponds to the `Fetch`, `OffsetCommit`, and `TxnOffsetCommit` Kafka requests. A `Write` operation for a `Topic` resource corresponds to the `Produce` and `AddPartitionsToTxn` Kafka requests.
+
+The following table lists the supported ACL permissions in {product}.
+
+NOTE: The resource identifiers `Starts with` and `Is` support the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topics in a Kafka instance.
```

Comment:
The wildcard is only valid for `Is` and not `Starts with`.

Reply:
Fixed. My bad, and that seems more logical. Thanks.